### PR TITLE
risk_model

### PR DESCRIPTION
**Pull request by yongqx**
I have added relevant definitions for risk factor, and start to write high-level structure for risk model.

### CODE REVIEW

File: docs/source/models/gbd2017/causes/tuberculosis/tuberculosis.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Active tuberculosis, also known as reactivation TB. It occurs when individual's
+immune system becomes weakened and is no longer able to contain the latent bacteria;
+then the TB bacteria become "active", overwhelm the immune system and cause a person
+to become ill. (UpToDate) In GBD, Active TB consist of 6 causes: Drug-susceptible
+TB, Multidrug-resistant TB, Extensively drug-resistant TB, HIV/AIDS - Drug-susceptible
+TB, HIV/AIDS - Multidrug-resistant TB, and HIV/AIDS - Extensively drug-resistant TB.
+
+TB Progression
+--------------
+Active TB disease developed within 2 years called fast progression, Individuals
```
Comment (aflaxman) [inline docs/source/models/gbd2017/causes/tuberculosis/tuberculosis.rst]: > Active TB disease developed within 2 years *is* called fast progression

File: docs/source/models/gbd2017/causes/tuberculosis/tuberculosis.rst [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+who do not have rapid progression are classified as having slow-progressing latent
+tuberculosis infection. With latent infection, individuals experience no adverse
+health effects and will not transmit Mycobacterium tuberculosis, but they face
+an ongoing risk of developing active tuberculosis through reactivation.
+
+TB cause model structure
+------------------------
+
+.. image:: TB_logical_structure.png
```
Comment (aflaxman) [inline docs/source/models/gbd2017/causes/tuberculosis/tuberculosis.rst]: Please change this to the left-to-right format that we used in the recent slide deck.

Comment (yongqx) [reply docs/source/models/gbd2017/causes/tuberculosis/tuberculosis.rst]: Will do!

**Comment by collijk** [comment]
This PR has been superceded by the set of PRs submitted by the rest of the research group.

**Pull request closed without merging**